frontend/appflowy_flutter/lib/workspace/presentation/settings/pages/account/account_deletion.dart

```diff
@@ -15,6 +15,8 @@ import 'package:flowy_infra_ui/flowy_infra_ui.dart';
 import 'package:flutter/material.dart';
 import 'package:toastification/toastification.dart';
 
+const _confirmText = 'DELETEMYACCOUNT';
```

Comment:
Should we add spaces?
```suggestion
const _confirmText = 'DELETE MY ACCOUNT';
```

Reply:
How about providing more acceptable confirmation texts? For example:

1. delete my account
2. deletemyaccount
3. DELETEMYACCOUNT
4. DELETE MY ACCOUNT